transport/kubernetes-jobmonitor/src/main/kotlin/JobHandler.kt

```diff
@@ -194,7 +185,19 @@ internal class JobHandler(
     private fun findPodsForJob(jobName: String): List<V1Pod> {
         val selector = "job-name=$jobName"
 
-        return api.listNamespacedPod(namespace, null, null, null, null, selector, null, null, null, null, false).items
+        return api.listNamespacedPod(
+            config.namespace,
```

Comment:
Can you put name parameters ? (and in other locations)
@mnonnenmacher always requests that :)

Reply:
We would here have to use commented parameter names as well, but I would rather not do this. I am not a big fan of the Kubernetes API with its numerous parameters. In these cases, however, it is quite clear which information is passed, since we have variables with speaking names between the *null* parameters.

Reply:
Fine with me. I was thinking about the last `false` though.